rosidl_generator_dotnet/resource/msg.cs.em

```diff
@@ -30,7 +30,23 @@ namespace @(ns)
 public class @(type_name) : IMessage {
     private static readonly DllLoadUtils dllLoadUtils;
 
-    public @(type_name)() {}
+    public @(type_name)()
+    {
+@[for member in message.structure.members]@
+@[    if isinstance(member.type, Array)]@
+// TODO: Array types are not supported
+@[    elif isinstance(member.type, AbstractSequence)]@
+// TODO: Sequence types are not supported
+@[    elif isinstance(member.type, AbstractWString)]@
+// TODO: Unicode types are not supported
+@[    elif isinstance(member.type, BasicType)]@
+@[    elif isinstance(member.type, AbstractString)]@
+        @(get_field_name(type_name, member.name)) = "";
+@[    else]@
+        @(get_field_name(type_name, member.name)) = new @(member.type.namespaces[0]).@(member.type.namespaces[1]).@(member.type.name)(); 
```

Comment:
I'm going to submit a PR to your branch to extend `get_dotnet_type` to handle namespaces -- that'll clean up this line and merge lines 212-215

Reply:
https://github.com/fmrico/ros2_dotnet/pull/4

Reply:
Merged